at_functional_test/test/commons.dart

```diff
@@ -103,8 +103,8 @@ Future<String> read({int maxWaitMilliSeconds = 5000}) async {
       result = _queue.removeFirst();
       // result from another secondary is either data or a @<atSign>@ denoting complete
       // of the handshake
-      if (result.startsWith('data:') ||
-          (result.startsWith('error:')) ||
+      if (result.contains('data:') ||
+          (result.contains('error:')) ||
```

Comment:
I still don't understand this. Why 'contains' instead of 'startsWith'?

Reply:
This i missed changing back to startsWith. Reverted back to startsWith and pushed the change.